### ISSUE TITLE
manager crash while grabbing logs 

### ISSUE THREAD

**Issue by Mtkmanjaro** [MuntashirAkon/AppManager#765]
- [x] I know what my device, OS and App Manager versions are
- [x] I know how to take logs
- [x] I know how to reproduce the issue which may not be specific to my device

**Describe the bug**
app manager crash whe grabbing logs for an specific app from running applications page

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to running applications
2. try to get log for an app
3. manager crash

**Expected behavior**
get the logs for the selected app

**Screenshots**
![Screenshot_20220527-082949_App Manager.png](https://user-images.githubusercontent.com/64289536/170723918-f14fd520-f463-42fb-979a-7ea6b1dedeb6.png)
manager crash while grabbing logs from here

**Crash logs**
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String io.github.muntashirakon.AppManager.logcat.struct.LogLine.getTag()' on a null object reference
    at io.github.muntashirakon.AppManager.logcat.LogViewerActivity.addToAutocompleteSuggestions(LogViewerActivity.java:1189)
    at io.github.muntashirakon.AppManager.logcat.LogViewerActivity.onNewLogsAvailable(LogViewerActivity.java:576)
    at io.github.muntashirakon.AppManager.logcat.LogViewerViewModel.lambda$sendNewLogs$2(LogViewerViewModel.java:165)
    at io.github.muntashirakon.AppManager.logcat.-$$Lambda$LogViewerViewModel$nugToyqfqdB56CfL5SZAm7EB0jg.run(Unknown Source:4)
    at android.os.Handler.handleCallback(Handler.java:938)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:201)
    at android.os.Looper.loop(Looper.java:288)
    at android.app.ActivityThread.main(ActivityThread.java:7839)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)

Device Info:
App version: 3.0.0-rc02
App version code: 404
Android build version: 1645012324
Android release version: 12
Android SDK version: 31
Android build ID: SQ1D.220205.003
Device brand: Xiaomi
Device manufacturer: Xiaomi
Device name: vince
Device model: Redmi 5 Plus
Device product name: lineage_vince
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: es-HN
In-App Language: auto
Mode: auto

**Comment by chr56** [comment]
Same exception, but it crashed when opening Log Viewer:

`java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String io.github.muntashirakon.AppManager.logcat.struct.LogLine.getTag()' on a null object reference`

stacktrace: 
```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String io.github.muntashirakon.AppManager.logcat.struct.LogLine.getTag()' on a null object reference
	at io.github.muntashirakon.AppManager.logcat.LogViewerActivity.addToAutocompleteSuggestions(LogViewerActivity.java:1189)
	at io.github.muntashirakon.AppManager.logcat.LogViewerActivity.onNewLogsAvailable(LogViewerActivity.java:576)
	at io.github.muntashirakon.AppManager.logcat.LogViewerViewModel.lambda$sendNewLogs$2(LogViewerViewModel.java:165)
	at io.github.muntashirakon.AppManager.logcat.-$$Lambda$LogViewerViewModel$nugToyqfqdB56CfL5SZAm7EB0jg.run(Unknown Source:4)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7666)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os
```

**Comment by MuntashirAkon** [comment]
Fixed in 7b8f5497f90c4fd067db91669628040d25ddd62b
